tests/test_utils.py

```diff
@@ -29,7 +29,9 @@
     ],
 )
 @pytest.mark.asyncio
-async def test_eval_answer(proposed, correct, question, eval_mode, expected):
+async def test_eval_answer(
+    proposed: str, correct: str, question: str | None, eval_mode: str, expected: bool
+) -> None:
```

Comment:
Feel free to ignore this, but this `expected` should be a float. I wanted to fix that in #149 , but ended up closing that PR.

Reply:
Nice point, done